README.md

```diff
@@ -1,87 +1,58 @@
 # Community tech tree for Workers & Resources
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Instalation
+A community tech tree replaces the Workers & Resources vanilla tech tree
+with one that is bigger, better, and managed by the modders community.
+The tree allows modders and content creators to lock their buildings
+behind the research in a scalable and synchronized way.
 
-TODO
-
-
-## How to add technologies
-
-
-
-$TYPE_TECHNICAL
-$COST 700
-
-$AVAILABLE
-
-$UNLOCK_RESEARCH engineering_1
-$UNLOCK_BUILDING mechanical_components_factory
-
-$NAME 11100
-$DESC 11101
-
```

Comment:
can we use $NAME_STR here?

Reply:
We can't :( This is also in an outdated version of the README.